lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -116,9 +116,33 @@ def void(authorization, options = {})
       end
 
       def verify(credit_card, options = {})
-        MultiResponse.run(:use_first_response) do |r|
-          r.process { authorize(100, credit_card, options) }
-          r.process(:ignore_result) { void(r.authorization, options) }
+        if options[:allow_card_verification] == true
+          options.delete(:allow_card_verification)
+          exp_month = credit_card.month.to_s
+          exp_year = credit_card.year.to_s
+          expiration = "#{exp_month}/#{exp_year}"
+          payload = {
+            creditCard: {
+              number: credit_card.number,
+              expirationDate: expiration,
+              cvv: credit_card.verification_value,
+              billing_address: {
+                postalCode: options[:billing_address][:zip]
+              }
+            }
+          }
```

Comment:
Also, out of curiosity, are there plans to add the `risk_data` or `external_vault` objects to this payload at some point? (It looks like maybe there are based on the conversation in Jira?)

Reply:
Great questions! Adding `risk_data` has been made into its own ticket in Jira for future work and we chatted about the `external_vault` situation in yesterday's standup. The end result was that since we are not always vaulting, it doesn't need to be included at this time.